### PR TITLE
Fetch event info

### PR DESCRIPTION
**Pull request by kathlynnguyenh11**

### CODE REVIEW

File: app/app.py [hunk 1]
```diff
@@ -187,18 +187,24 @@ def api_events():
...
-                    "period": event.period,
-                }
-                for event in contact.events
-            ]
-            for contact in user.contacts
-        }
+        date = request.args.get('date', '')
+        if date == '':
+            return ("Error: No id field provided. Please specify an id.", 400)
+        date_time_obj = datetime.strptime(date, "%Y-%m-%d")
```
Comment (peppermintpatty5) [inline app/app.py]: Careful, this should be `datetime.datetime.strptime`.

**Review by peppermintpatty5 — DISMISSED** [review]
I think you should think about what you want the API to do for getting events. If there is no date in the query, then perhaps return all the user's events. If there is a date, in what structure should the events be returned? Right now, I would not be sure how to use this API.

**Comment by kathlynnguyenh11** [comment]
> I think you should think about what you want the API to do for getting events. If there is no date in the query, then perhaps return all the user's events. If there is a date, in what structure should the events be returned? Right now, I would not be sure how to use this API.

So what this API does is to fetch event information when I click on a date so I should only return the events of that specific user on that specific date. I hope that makes sense.

**Review by peppermintpatty5 — APPROVED** [review]
Can fix the annoying timezone conversion issues later :roll_eyes:

**Pull request merged**